### PR TITLE
NFT-387 feat: introducing Community page

### PR DESCRIPTION
**Pull request by cnasc**
localhost_3000_network_ethereum_community

Update: when merged this will show up on prod

### CODE REVIEW

**Comment by jringenberg** [comment]
@cnasc Can you update the text with this link to the bridge?
the link is: https://app.optimism.io/bridge

image

File: components/CommunityActivity/CommunityActivity.tsx [hunk 1]
```diff
@@ -21,15 +22,22 @@ export function CommunityActivity({ account }: CommunityActivityProps) {
...
-      </ol>
+      <Fieldset legend="✨ XP Earned">
+        {scoreChanges.length === 0 && (
+          <span>No XP earned by this address yet.</span>
+        )}
+        {scoreChanges.length > 0 && (
+          <ol className={styles.list}>
+            {scoreChanges.map((event) => (
+              <li data-testid={`event-${event.id}`} key={event.id}>
+                {event.category.name} 1XP{' '}
```
Comment (adamgobes) [inline components/CommunityActivity/CommunityActivity.tsx:33]: ok first thing i notice, and i forgot to mention this so its on me, is that the `CategoryScoreChange` entity has an oldScore and newScore field

File: lib/constants.ts [hunk 2]
```diff
@@ -24,6 +24,8 @@ export const BUNNY_IMG_URL_MAP = {
 };
 
 export const COMMUNITY_NFT_CONTRACT_ADDRESS =
-  '0x7887bad2a088027dabce45c81229092a4112f622';
+  '0x63a9addF2327A0F4B71BcF9BFa757E333e1B7177';
+
+// TODO: optimism on launch
```
Comment (adamgobes) [inline lib/constants.ts:29]: can remove this todo (:

**Review by adamgobes — APPROVED** [review]
lgtm :shipit:

**Pull request merged**